src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/MediaApplicationController.java

```diff
@@ -111,9 +114,20 @@ public ResponseEntity<MediaApplication> updateApplication(@PathVariable UUID id,
             mediaApplicationService.updateApplication(id, status));
     }
 
+    @ApiResponse(responseCode = OK_ERROR_CODE, description = MEDIA_APPLICATION)
```

Comment:
Sorry just thinking more over lunch, I do think this should be part of the update application service for rejection.

My thought is you would never want an application to be rejected without this email being sent - therefore to reduce that extra call and prevent that scenario from happening, it feels better to do it in one.